testpilot/frontend/static-src/styles/modules/_main-header.scss

```diff
@@ -7,6 +7,8 @@
     font-weight: 700;
     margin: 10px 0;
 
+    a { cursor: pointer; }
```

Comment:
Hmm, good question @6a68 . Looking at it again, I don't think it's necessary.  When I tested this styling earlier, I had thought my cursor stayed an arrow icon instead of the pointer hand image. Now however, I can see that the cursor has pointer styling whether I declare it or not. I think initially my browser was slow to adjust to my cursor hovering over the link, and so I thought it wasn't displaying the style it needed.

Should I do a separate PR without this style declaration?